extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/SyncHttpClientConfig.java

```diff
@@ -0,0 +1,214 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SyncHttpClientConfig {
+
+    /**
+     * Type of the sync HTTP client implementation
+     *
+     * <p>
+     * Accepted values are `URL` or `APACHE`
+     */
+    @ConfigItem(defaultValue = "url")
+    public SyncClientType type;
+
+    /**
+     * The maximum amount of time to establish a connection before timing out.
+     *
+     * <p>
+     * Used by `URL` and `APACHE` clients
+     */
+    @ConfigItem(defaultValue = "2S")
+    public Duration connectionTimeout;
+
+    /**
+     * The amount of time to wait for data to be transferred over an established, open connection before the connection is
+     * timed
+     * out.
+     * 
+     * <p>
+     * Used by `URL` and `APACHE` clients
+     */
+    @ConfigItem(defaultValue = "30S")
+    public Duration socketTimeout;
+
+    /**
+     * Apache HTTP client additional configuration
+     */
+    @ConfigItem(name = ConfigItem.PARENT)
+    public ApacheHttpClientConfig apacheHttp;
+
+    @ConfigGroup
+    public static class ApacheHttpClientConfig {
+
+        /**
+         * The amount of time to wait when acquiring a connection from the pool before giving up and timing out.
+         * 
+         * <p>
+         * Used by `APACHE` client only
```

Comment:
Then you could drop this. I will tweak the documentation with some sections and it will be pretty much OK I think.

Reply:
will drop it